rpc/mercury.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+option go_package = " github.com/smartcontractkit/chainlink-data-streams/rpc";
+
+package pb;
+
+service Mercury {
+    rpc Transmit(TransmitRequest) returns (TransmitResponse);
+    rpc LatestReport(LatestReportRequest) returns (LatestReportResponse);
+}
+
+message TransmitRequest {
+    bytes payload = 1;
+    uint32 reportFormat = 2;
+}
+
+message TransmitResponse {
+    int32 code = 1;
+    string error = 2;
+}
+
+message LatestReportRequest {
+    bytes feedId = 1;
+}
+
+message LatestReportResponse {
+    string error = 1;
+    Report report = 2;
+}
```

Comment:
We really don't need to support these, would recommend dropping it from here.

Reply:
We need to keep them to be reusable for the current v0.3 plugin until the migration is done. For the LLO please narrow the local client interface to the  interface Transmit call only.